src/migrations/versions/2024_05_30_1200-f84a0ed81bdc_init.py

```diff
@@ -22,7 +22,7 @@
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     op.create_table(
-        "camera",
+        "cameras",
```

Comment:
reminder that we'll need to test extensively the migration script or clean it at some point 😅